codespan-reporting/src/diagnostic.rs

```diff
@@ -16,7 +16,7 @@ use std::ops::Range;
 /// assert!(Severity::Warning > Severity::Note);
 /// assert!(Severity::Note > Severity::Help);
 /// ```
-#[derive(Copy, Clone, PartialEq, Hash, Debug)]
+#[derive(Copy, Clone, PartialEq, Hash, Debug, Eq)]
```

Comment:
Good call, thanks :)

Reply:
Thanks, this is a handy one! Could also maybe do with `Ord`?